packages/server/src/api/controllers/static/index.js

```diff
@@ -71,7 +71,7 @@ exports.uploadFile = async function (ctx) {
 
     return prepareUpload({
       file,
-      s3Key: `assets/${ctx.appId}/attachments/${processedFileName}`,
+      s3Key: `/${ctx.appId}/attachments/${processedFileName}`,
```

Comment:
Not sure if the leading slash here in the key might cause issues - but I don't think it's required

Reply:
Good catch, wasn't there before so shouldn't be there now!